openvinotoolkit/datumaro
# Enable stream exporters: VOC, YOLO, Datumaro, and COCO data format

### Summary
 - Ticket no. 114762

### How to test
Please see our OneNote page: Datumaro -> Stream import/export -> Stream exporter

### Checklist


### License

  Feel free to contact the maintainers if that's a concern.

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```
